test/message-generator/test/translation-proxy-old-share.json

```diff
@@ -0,0 +1,105 @@
+{
+    "version": "1",
+    "doc": [
+        "This test does",
+        "Mock an Upstream",
+        "Run tProxy",
+        "tProxy receives NewExtendedMiningJob (with job_id = 1)",
+        "tProxy receives mining.configure, mining.subscribe, mining.authorize",
+        "tProxy sends mining.notify",
+        "tProxy receives mining.submit (old/invalid share with job_id=0)"
+    ],
+    "sv1_messages": [
+        {
+            "message": {
+                "id": 1,
+                "method": "mining.authorize",
+                "params": ["username", "password"]
+            },
+            "id": "mining.authorize"
+        },
+        {
+            "message": {
+                "id": 2,
+                "method": "mining.submit",
+                "params": ["username", "0", "0000000000000000", "641577b0", "7a600640"]
+            },
+            "id": "mining.submit"
+        }
+    ],
+    "frame_builders": [
+    ],
+    "actions": [
+        {
+            "message_ids": ["mining.authorize"],
+            "results": [
+                {
+                    "type": "match_message_id",
+                    "value": 1
+                }
+            ]
+        },
+        {
+            "message_ids": ["mining.submit"],
+            "results": [
+                {
+                    "type": "match_message_field",
+                    "value": [
+                        "mining.submit",
+                        [
+                            [
+                                "result",
+                                true
+                            ]
+                        ]
+                    ]
+                }
+            ],
+            "actiondoc": "Checks that the mining.submit request (with wrong job_id) generates a response with false, indicating that the share was rejected"
```

Comment:
this is not what the test does

Reply:
78749a06cc81d8f43bb1b2e510bd544920e12343